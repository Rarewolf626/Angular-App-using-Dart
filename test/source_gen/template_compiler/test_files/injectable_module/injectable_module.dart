import 'package:angular2/angular2.dart';

@InjectorModule(providers: const [])
class EmptyModule {}

@InjectorModule(providers: const [EmptyModule])
class TestModule {}
