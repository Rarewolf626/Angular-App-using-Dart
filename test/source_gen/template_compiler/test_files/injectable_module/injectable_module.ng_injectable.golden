// GENERATED CODE - DO NOT MODIFY BY HAND

// **************************************************************************
// Generator: TestInjectableExtractor
// Target: library
// **************************************************************************

final String output = """[
  {
    "class": "InjectorModule",
    "name": "EmptyModule",
    "moduleUrl": "asset:angular2/test/source_gen/template_compiler/test_files/injectable_module/injectable_module.dart",
    "prefix": null,
    "isHost": false,
    "value": null,
    "diDeps": [],
    "providers": [],
    "injectable": true
  },
  {
    "class": "InjectorModule",
    "name": "TestModule",
    "moduleUrl": "asset:angular2/test/source_gen/template_compiler/test_files/injectable_module/injectable_module.dart",
    "prefix": null,
    "isHost": false,
    "value": null,
    "diDeps": [],
    "providers": [],
    "injectable": true
  }
]""";
